src/item.cpp

```diff
@@ -2237,12 +2268,22 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
-    if( t == nullptr ) {
+    float pockets = 1;
+    auto t = find_armor_data();
+    if( t == nullptr )
         return 0;
-    }
+
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
+    int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
+
+    if (item::item_tags.count("pocketed") > 0){
+    pockets = 1.5;
+        if (result > 0)
+            return result * pockets;
+        else return result + pockets + 3.5; // Can I just use return 5?
```

Comment:
OK, I added the Pockets to my pair of jeans.  Result: +1 storage, so I'm worse off enhancing the jeans than I'd be enhancing leather pants or a nanoskirt.

Suggestion: perhaps this should increase the storage (and the encumbrance!) by the target clothing's coverage percentage, minimum increase on each = 1.  (So you can add even more pockets to your cargo pants, but they'll get in the way; and no fair giving a miniskirt storage 5.)  Can't be used on FANCY or SUPER_FANCY gear, as the pockets would just undermine the look.